packages/components/organisms/f-cookie-banner-static/gulpfile.js

```diff
@@ -12,7 +12,7 @@ const PATHS = {
     vueSrcFolder: './src',
     tempDistFolder: './.tmp-dist',
     tempVueFolder: './.tmp-vue',
-    rootDistFolder: '../dist/static'
+    rootDistFolder: '../f-cookie-banner/dist/static'
```

Comment:
Should this be pointing at the `f-cookie-banner` folder, or `f-cookie-banner-static`?